launcher/settingsView/csettingsview_moc.cpp

```diff
@@ -163,6 +164,24 @@ void CSettingsView::fillValidScalingRange()
 
 #ifndef VCMI_MOBILE
 
+static QStringList getAvailableRenderingDrivers()
+{
+	SDL_Init(SDL_INIT_VIDEO);
```

Comment:
you must link SDL lib to the launcher target in CMake

Reply:
We actually already use SDL in launcher and it works. So I presume already linked with SDL

Reply:
yep, already in:
```
if (NOT APPLE_IOS AND NOT ANDROID)
	target_link_libraries(vcmilauncher SDL2::SDL2)
endif()
```

Reply:
Nice, forgot about that. Probably to get supported resolutions.